pkg/lib/bundle/utils_test.go

```diff
@@ -70,8 +70,7 @@ func clearDir(dir string) {
 	for _, item := range items {
 		if item.IsDir() {
 			continue
-		} else {
-			os.Remove(filepath.Join(dir, item.Name()))
 		}
+		os.Remove(filepath.Join(dir, item.Name()))
```

Comment:
There's already a continue in that first conditional check, so we don't need to wrap file removal logic and can remove that branch entirely.